File: Modul 6/Bahasa Python/PRAK602_2310817120001_DinaIzzatiElfadheya.py
print("Input")

jumlahZetsu = int(input())

zetsuPutih = []
zetsuPutih = list(map(int, input().split()))

for x in range(jumlahZetsu):
    zetsuPutih[x] *= (x + 1)

print("\nOutput")
for x in range(jumlahZetsu):
    print(zetsuPutih[x], end=" ")

File: Modul 6/Bahasa C/PRAK602_2310817120001_DinaIzzatiElfadheya.c
#include <stdio.h>
int main() {
    int jumlahZetsu;

    printf("Input\n");
    scanf("%d", &jumlahZetsu);

    int zetsuPutih[jumlahZetsu];

    for (int x = 0; x < jumlahZetsu; x++) {
        scanf("%d", &zetsuPutih[x]);
    }
    for (int x = 0; x < jumlahZetsu; x++) {
        zetsuPutih[x] *= (x + 1);
    }
    printf("\nOutput\n");
    
    for (int x = 0; x < jumlahZetsu; x++) {
        printf("%d", zetsuPutih[x]);
        printf(" ");
    }
    return 0;
}
